setup.cfg

```diff
@@ -28,6 +28,7 @@ test = pytest
 
 [coverage:run]
 branch = True
+relative_files = True
```

Comment:
makes sense. I just don't know if we have a bunch of .coverage.id files which we have to combine